appwrite.template.json

```diff
@@ -10,6 +10,384 @@
+                    "status": "available",
+                    "error": "",
+                    "required": true,
+                    "array": false,
+                    "format": "",
+                    "default": null
+                },
+                {
+                    "key": "products",
+                    "type": "string",
+                    "status": "available",
+                    "error": "",
+                    "required": false,
+                    "array": false,
+                    "size": 65535,
+                    "default": null
+                },
+                {
+                    "key": "tax",
+                    "type": "double",
+                    "status": "available",
+                    "error": "",
+                    "required": false,
+                    "array": false,
+                    "min": 0,
+                    "max": 1,
+                    "default": 0
+                },
+                {
+                    "key": "total",
+                    "type": "double",
+                    "status": "available",
+                    "error": "",
+                    "required": true,
+                    "array": false,
+                    "min": 0,
+                    "max": null,
+                    "default": null
+                },
+                {
+                    "key": "status",
+                    "type": "string",
+                    "status": "available",
+                    "error": "",
+                    "required": false,
+                    "array": false,
+                    "elements": [
+                        "draft",
+                        "pending",
+                        "paid"
+                    ],
+                    "format": "enum",
+                    "default": "draft"
+                },
+                {
+                    "key": "subject",
+                    "type": "string",
+                    "status": "available",
+                    "error": "",
+                    "required": true,
+                    "array": false,
+                    "size": 250,
+                    "default": null
+                },
+                {
+                    "key": "note",
+                    "type": "string",
+                    "status": "available",
+                    "error": "",
+                    "required": true,
+                    "array": false,
+                    "size": 250,
+                    "default": null
+                },
+                {
+                    "key": "terms",
+                    "type": "string",
+                    "status": "available",
+                    "error": "",
+                    "required": true,
+                    "array": false,
+                    "size": 250,
+                    "default": null
+                }
+            ],
+            "indexes": []
+        },
+        {
+            "$id": "saved-products",
```

Comment:
The saved products is kind of like a template of products. You select the saved product and it copies the description, quantity, price, and total from the saved product into the invoice.